docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,436 @@
+signed or unsigned. Details on what constitutes a valid staking transaction
+can be found [here](./staking-script.md#staking-transaction).
+One can create a staking transaction by either:
+  * Using our [Golang BTC staking library](../btcstaking)
+  * Using our [TypeScript BTC staking library](https://github.com/babylonlabs-io/btc-staking-ts)
+  * Implementing the transaction generation themselves using the 
+  [staking script specification](./staking-script.md) as a reference.
+* `staking_tx_inclusion_proof` (Optional): 
+This is a Merkle proof showing that the staking transaction is included in the 
+Bitcoin chain. This field is optional but recommended.
+Below is the proto message for the `inclusion_proof` field with the fields:
+  * `txIdx`: The index of the transaction within a Bitcoin block. This indicates
+    the transaction's position in the list of transactions in that block.
+  * `blockHash`: The hash of the Bitcoin block containing the transaction.
+    This uniquely identifies the block in the Bitcoin blockchain.
+```proto
+message InclusionProof {
+    // key is the position (txIdx, blockHash) of this tx on BTC blockchain
+    babylon.btccheckpoint.v1.TransactionKey key = 1;
+    // proof is the Merkle proof that this tx is included in the position in `key`
+    bytes proof = 2;
+}
+```
+If not provided during submission, it must be submitted later using the 
+`MsgAddBTCDelegationInclusionProof` message.
+* `slashing_tx`: 
+The slashing transaction in hex format. Generated during both staking and 
+unbonding phases, this transaction spends the staking or unbonding transaction 
+in cases of protocol violations such as double-signing. To see exactly what 
+constitutes a valid slashing transaction, see the 
+[`Slashing Transaction` section](./staking-script.md#slashing-transaction).
+A valid signature for this transaction must be included in the 
+`slashing_tx_sig` field.
+To generate or obtain the slashing transaction, you can use the libraries 
+mentioned below:
+  * Using our [Golang BTC staking library](../btcstaking)
+  * Using our [TypeScript BTC staking library](https://github.com/babylonlabs-io/btc-staking-ts)
+  * Implementing the transaction generation themselves using the 
+    [staking script specification](./staking-script.md) as a reference.
+* `delegator_slashing_sig`: 
+The signature for the `slashing_tx`, which is crucial for authorizing the 
+transaction in the event of a protocol violation, such as double-signing. 
+The signature, generated with the private key for the staker's secp256k1 
+public key (`btc_pk`), should be compatible with ECDSA or Schnorr (BIP340).
+* `unbonding_time`: 
+The timelock period for unbonding, measured in Bitcoin blocks, indicating 
+how long funds remain locked. It should be an unsigned integer (uint32) and 
+ensure that any `unbonding_time` you choose is more than that specified 
+by the `min_unbonding_time_blocks` parameter in the genesis file.
```

Comment:
It's not in the genesis file but in the Babylon parameters. Do you think that it makes to have a `3.2. Babylon chain BTC Staking parameters` section which explains the parameters of the `x/btcstaking` module, what they mean, and how they can be retrieved? As a lot of this data relates to the Babylon parameters.